python/tank_vendor/shotgun_authentication/ui/login_dialog.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) 2013 Shotgun Software Inc.
```

Comment:
Our normal convention with regards to ui files is that there is a resources folder which contains UI files and resource files (icons etc). We then have a `ui` folder on the code side which contains the corresponding, auto generated files (here for example: https://github.com/shotgunsoftware/tk-multi-about/tree/master/python/tk_multi_about/ui ). I think it would probably be good to follow that convention here too just for simplicity. I guess that means we move this file up one level in the folder structure since it wasn't auto generated.